posthog-web/src/posthog-web.ts

```diff
@@ -9,6 +9,11 @@ import { PostHogStorage, getStorage } from './storage'
 import { version } from '../package.json'
 import { PostHogOptions } from './types'
 
+export function _getWindow(): Window | undefined {
+  const _window: Window | undefined = typeof window !== 'undefined' ? window : undefined
+  return _window
```

Comment:
very nit-picky but `_window` can be in-lined here
```suggestion
  return  typeof window !== 'undefined' ? window : undefined
```